### PR TITLE
feat(inputs.mongodb): Add metrics about files currently open and currently active data handles

### PR DESCRIPTION
**Pull request by DrissTM**
resolve: #7895

This PR is adding two metrics in the mongodb plugin:
- wt_connection_files_currently_open (can be obtained with the following command: `db.serverStatus()["wiredTiger"]["connection"]["files currently open"]`)
- wt_data_handles_currently_active (can be obtained with the following command: `db.serverStatus()["wiredTiger"]["data-handle"]["connection data handles currently active"]`)

These metrics are bound to the number of actively used collections/indexes (the more collections/indexes you actively use the more `files currently open`/`connection data handles currently active` you'll have), high values of these can cause checkpoints to hold the schemaLock longer than usual and result in global database locks every 60 seconds (see #7895 for more details and link to mongodb jira issue)

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

**Comment by srebhan** [comment]
@DrissTM can you please resolve the merge conflict. Will review afterwards.

**Comment by srebhan** [comment]
@DrissTM are you still interested in this PR?

**Comment by DrissTM** [comment]
hi @srebhan I'm working again on this PR, there's one last conflict to solve and it should be ready for review

File: plugins/inputs/mongodb/README.md [hunk 1]
```diff
@@ -299,6 +301,11 @@ by running Telegraf with the `--debug` argument.
...
...720i,tcmalloc_spinlock_total_delay_ns=9767500i,tcmalloc_thread_cache_free_bytes=1222608i,tcmalloc_total_free_bytes=1797400i,tcmalloc_transfer_cache_free_bytes=294656i,total_available=0i,total_created=0i,total_docs_scanned=43595i,total_in_use=0i,total_keys_scanned=130805i,total_refreshing=0i,total_tickets_reads=128i,total_tickets_writes=128i,ttl_deletes=0i,ttl_deletes_per_sec=0i,ttl_passes=0i,ttl_passes_per_sec=0i,update_command_failed=0i,update_command_total=43595i,updates=43595i,updates_per_sec=372i,uptime_ns=60023000000i,version="3.6.17",vsize_megabytes=1048i,wtcache_app_threads_page_read_count=108i,wtcache_app_threads_page_read_time=25995i,wtcache_app_threads_page_write_count=0i,wtcache_bytes_read_into=2487250i,wtcache_bytes_written_from=74i,wtcache_current_bytes=5014530i,wtcache_internal_pages_evicted=0i,wtcache_max_bytes_configured=505413632i,wtcache_modified_pages_evicted=0i,wtcache_pages_evicted_by_app_thread=0i,wtcache_pages_queued_for_eviction=0i,wtcache_pages_read_into=139i,wtcache_pages_requested_from=699135i,wtcache_server_evicting_pages=0i,wtcache_tracked_dirty_bytes=4797426i,wtcache_unmodified_pages_evicted=0i,wtcache_worker_thread_evictingpages=0i 1586379818000000000
```
Comment (sspaink) [inline plugins/inputs/mongodb/README.md:310]: Should this original example output be removed? Looks like the one you added replaces this one.

**Review by sspaink — CHANGES_REQUESTED** [review]

**Comment by sspaink** [comment]
@DrissTM were these values already been grabbed but just ignored by the plugin? Wondering if adding these metrics would cause extra calls for every user or if it makes use of an existing request.

File: plugins/inputs/mongodb/README.md (hunk 1 above)
Comment (DrissTM) [reply plugins/inputs/mongodb/README.md]: yes, that's the merge conflict I missed when updating my branch, I fixed it in [b47a434](https://github.com/influxdata/telegraf/pull/7896/commits/b47a434e12938eccbd56a906ef41f239beaffbfa)

**Pull request merged**